src/webpack.js

```diff
@@ -213,7 +213,7 @@ module.exports = async ({ entrypoint, serviceName = 'test-service', ...options }
     module: {
       rules: [
         {
-          loader: 'babel-loader',
+          ...babelLoaderConfig,
```

Comment:
I don't think this is the right change to make. The rule defined here is made to convert async methods into generators so that X-Ray tracing will work correctly. Since we want X-Ray tracing to work correctly for all code I don't think that the engines filter should be included here